taichi/runtime/llvm/llvm_context.cpp

```diff
@@ -95,6 +95,11 @@ TaichiLLVMContext::TaichiLLVMContext(CompileConfig *config, Arch arch)
     llvm::InitializeNativeTargetAsmParser();
 #endif
   } else if (arch == Arch::dx12) {
+    // FIXME: Must initialize these before initializing Arch::dx12
```

Comment:
OOC: could you explain a bit why it wasn't required before this PR? :P Thanks!

Reply:
Because we initialized the TaichiLLVMContext of CPU before initializing the context of the backend specified by config.arch regardless of what the backend is before. We don't initialized it now if the backend is not CPU.